langchain_benchmarks/tool_usage/tasks/multiverse_math.py

```diff
@@ -193,9 +193,9 @@ def get_environment() -> ToolUsageEnvironment:
         "expected_steps": ["log", "multiply"],
     },
     {
-        "question": "calculate 101 to the power of 0.5 to 4 digits of precision",
-        "answer": round(power(101, 0.5), 4),
-        "expected_steps": ["power", "round"],
+        "question": "calculate 101 to the power of 0.5",
+        "answer": power(101, 0.5),
+        "expected_steps": ["power"],
```

Comment:
this was incorrect in terms of expected steps, since "round" wasn't a function. Also removing the request for digits of precision since the `question` isn't an input to the evaluator